worker-workflow/src/test/resources/action.json

```diff
@@ -1,13 +0,0 @@
-{
-    "name": "family_hashing",
-    "queueName": "dataprocessing-family-hashing-in",
-    "customData": {
-        "tenantId": "tenantId"
-    },
-    "scripts": [
-        {
-            "name": "recordProcessingTimes.js",
-            "script": "function onProcessTask(e) {\n  var startTime \u003d new Date();\n  e.rootDocument.getField(\"LONG_FAMILY_HASHING_START_TIME\").set(startTime.getTime());\n  e.rootDocument.getField(\"ENRICHMENT_TIME\").set(Math.round(startTime.getTime() / 1000));\n}\nfunction onAfterProcessTask(e) {\n  var endTime \u003d new Date();\n  e.rootDocument.getField(\"LONG_FAMILY_HASHING_END_TIME\").set(endTime.getTime());\n}\nfunction onError(e) {\n  var failedTime \u003d new Date();\n  e.rootDocument.getField(\"LONG_FAMILY_HASHING_FAILED_TIME\").set(failedTime.getTime());\n}\n"
-        }
-    ]
-}
```

Comment:
Why is this file being removed?  Is that to do with this Jira?

Reply:
Since I am passing the actions to the workflow script in a different manner (see [here](https://github.com/CAFDataProcessing/worker-workflow/blob/f9cbe0a44adc0ebcecd93bff75b7b1b029b3b541/worker-workflow/src/test/java/com/github/cafdataprocessing/workflow/WorkflowControlTest.java#L1159)), the file was not needed anymore.